crates/ibc/src/core/ics02_client/height.rs

```diff
@@ -159,6 +159,8 @@ pub enum HeightError {
     },
     /// attempted to parse an invalid zero height
     ZeroHeight,
+    /// the height(`{raw_height}`) is not valid, this format must be used: \[revision_number\]-\[revision_height\]
+    InvalidHeight { raw_height: String },
```

Comment:
Would you rename this to smt like `InvalidFormat`? 
This can a bit confusing given the other [InvalidHeight](https://github.com/cosmos/ibc-rs/blob/522aef1a68b80966e8e62fdacd5e3b05818f2784/crates/ibc/src/core/ics02_client/error.rs#L73) we already have under the `ClientError` enum